projects/1.18/assets/tiny-redstone/tinyredstone/lang/zh_cn.json

```
@@ -1 +1,96 @@
+  "item.tinyredstone.tiny_stone_button": "微型石头按钮",
+  "message.item.tinyredstone.tiny_stone_button": "激活时向4面及底部输出强度为15的红石信号，持续10红石刻。",
+  "item.tinyredstone.silicon": "硅",
+  "item.tinyredstone.silicon_compound": "硅化合物",
+  "item.tinyredstone.tiny_observer": "微型侦测器",
+  "message.item.tinyredstone.tiny_observer": "检测到所面向网格单元或方块状态发生变化时，向后方输出红石脉冲。\n当其所面向方块位于世界中时，行为与原版侦测器相同。\n当其所面向网格单元位于面板中时，将检测网格单元的运动，放置，移除及红石信号改变。\n可被微型活塞推动。",
+  "item.tinyredstone.blueprint": "微型红石蓝图",
+  "item.tinyredstone.tiny_super_repeater": "微型超级中继器",
+  "message.item.tinyredstone.tiny_super_repeater": "与微型中继器类似，右击将打开一个GUI，其中可以设置从0到100红石刻（可在配置文件中配置）的延迟。\n该元件可作为0t二极管或长延迟时钟电路使用。\n过长红石线可能会导致溢出。上限可在配置文件中设置。",
+  "item.tinyredstone.tiny_lever": "微型拉杆",
+  "message.item.tinyredstone.tiny_lever": "激活时向4面及底部输出红石信号。\n右击切换开/关。",
+  "item.tinyredstone.tiny_redstone_bridge": "微型红石桥",
+  "message.item.tinyredstone.tiny_redstone_bridge": "与红石粉类似，但一侧接收到的信号只能输出到对侧。\n这将允许信号交叉传输。",
+  "item.tinyredstone.light_panel_cover": "透光面板罩",
+  "message.item.tinyredstone.light_panel_cover": "隐藏电路并阻止渲染。\n允许光照穿过。\n使用方块切碎机来制作与原方块材质相同的面板罩。",
+  "item.tinyredstone.dark_panel_cover": "遮光面板罩",
+  "message.item.tinyredstone.dark_panel_cover": "隐藏电路，阻止渲染及客户端同步。\n不允许光照穿过。\n使用方块切碎机来制作与原方块材质相同的面板罩。",
+  "item.tinyredstone.trim_panel_cover": "剪裁面板罩",
+  "message.item.tinyredstone.trim_panel_cover": "根据电路情况自动调整大小，隐藏电路，阻止渲染及客户端同步。\n不允许光照穿过。",
+  "item.tinyredstone.tiny_color_selector": "颜色选择器",
+  "item.tinyredstone.tiny_note_block": "微型音符盒",
+  "message.item.tinyredstone.tiny_note_block": "右击调整音高。手持微型元件扳手右击选择乐器。\n微型音符盒或其毗邻的方块（由于半连接性）接收到红石脉冲时，将奏出音符。\nThe One Probe及WAILA模组可以显示当前乐器及音高。\n可被微型活塞推动。",
+  
+  "block.tinyredstone.block_chopper": "方块切碎机",
+  "message.block.tinyredstone.block_chopper": "使用绝大多数完整方块制作微型方块及面板罩。不需要硅！",
+  
+  "tinyredstone.tooltip.press_shift": "--按住Shift查看更多信息--",
+  "tinyredstone.close": "关闭",
+  "tinyredstone.yes": "是",
+  "tinyredstone.gui.repeater.msg": "设置红石刻延迟（10/秒）",
+  "tinyredstone.gui.repeater.msg2": "使用按钮或滚轮以调节延迟。",
+  "tinyredstone.gui.repeater.msg3": "按住Shift键滚动以按秒调节。",
+  "tinyredstone.gui.tinyblock.msg": "选择一种颜色",
+  "tinyredstone.gui.blueprint.msg": "蓝图",
+  "message.item.blueprint.full": "§7右击将电路放置在面板或世界中。\n对空气右击来将电路导出为JSON文件。\n§d需要以下材料：§r",
+  "message.item.blueprint.empty": "§7右击一个电路将其保存在蓝图中。\n对空气右击来从JSON文件中导入电路。§r",
+  "tinyredstone.import": "导入",
+  "tinyredstone.export": "导出",
+  "tinyredstone.cancel": "取消",
+  "tinyredstone.choose_file": "选择JSON文件",
+  "tinyredstone.save_file": "将JSON文件另存为...",
+  "tinyredstone": "微型红石",
+  "tinyredstone.gui.crash.msg": "该面板已崩溃。\n请检查你的日志文件。\n如有可能，请向作者反馈此次崩溃。\n现在你只能移除元件。\n点击‘重启’来尝试重新启用面板。",
+  "tinyredstone.gui.overflow.msg": "该面板上的电路已溢出。\n0t红石线可能过长。\n尝试增加延迟，或在配置文件中增加红石最大长度。\n点击‘重启’来清除此错误提示。",
+  "tinyredstone.enable": "重启",
+  "tinyredstone.gui.clearpanel.msg": "移除所有元件？",
+  
+  "tinyredstone.noteblock.bass": "贝斯",
+  "tinyredstone.noteblock.snare": "小军鼓",
+  "tinyredstone.noteblock.hat": "击鼓沿",
```

```suggestion
  "tinyredstone.noteblock.hat": "踩镲",
```
https://zh.minecraft.wiki/w/%E9%9F%B3%E7%AC%A6%E7%9B%92#%E4%B9%90%E5%99%A8